src/importexport/musicxml/tests/tst_mxml_io.cpp

```diff
@@ -404,6 +412,31 @@ void TestMxmlIO::mxmlMscxExportTestRefBreaks(const char* file)
     delete score;
 }
 
+void TestMxmlIO::mxmlMscxExportTestRefInvisibleElements(const char* file)
```

Comment:
@dmitrio95 do you have instructions how to run this test? I can't figure it out with MSVC!

Reply:
I have only launched this in Linux so I cannot tell for sure what may be needed for MSVC. However MuseScore seems to make use of `ctest` which should probably work on different platforms. So running the following from the build directory should work:
```
ctest -R iex_musicxml_tests
```
or, to get a more verbose output,
```
ctest --verbose -R iex_musicxml_tests
```

Another option is to run the relevant test executable manually. For me (with Linux and Ninja build system) the executable for this test is generated at
```
build.dir/src/importexport/musicxml/tests/iex_musicxml_tests
```
where `build.dir` is the name of the directory where I launched the build. Therefore I can launch MusicXML tests just by running this executable.

Maybe some of these options can work with MSVC as well.

Reply:
@wizofaus, if you just want to check the output then you can run the artifact from the GitHub Actions build. I think it includes invisible elements by default (but marks them invisible with `print-object="no"`), but if you want to try excluding them then you can edit the [preferences INI file](https://musescore.org/en/handbook/3/revert-factory-settings#Via_command_line), which is located at:

```
Windows: %APPDATA%\MuseScore\MuseScore4Development.ini
Linux: ${XDG_CONFIG_HOME:-~/.config}/MuseScore/MuseScore4Development.ini
```
You need to add this line:

```ini
export/musicXML/exportInvisibleElements=false
```

It's a bit different on macOS, where the preferences are stored in a binary `plist` format.

```
macOS: ~/Library/Preferences/org.musescore.MuseScore4Development.plist
```

You can use XCode to edit the `plist` file. Add a new row inside the Root object:

Key | Type | Value
:---|:---|:---
export.musicXML.exportInvisibleElements | Boolean | 0

Reply:
I was really wanting to understand how to debug unit tests in general in MSVC.  I can't see anyway of even compiling them let alone running them (I tried setting up a project manually to do so but eventually got stuck with no having the .moc files being generated).
I had thought they were just not being used currently but obviously other devs are able to develop/execute/debug unit tests and I really would like to be able to do myself, I'm not used to submitting PRs with changes and no associated unit tests!